### PR TITLE
Change DotNetCore installation default package type

### PR DESCRIPTION
**Pull request by bishal-pdMSFT**

### CODE REVIEW

File: Tasks/DotNetCoreInstaller/task.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
             "name": "packageType",
             "type": "pickList",
             "label": "Package to install",
-            "defaultValue": "runtime",
+            "defaultValue": "sdk",
```
Comment (sachinma) [inline Tasks/DotNetCoreInstaller/task.json]: does the default version value also need to change?

Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreInstaller/task.json]: No..same version exists for SDK as well.

Comment (bishal-pdMSFT) [inline Tasks/DotNetCoreInstaller/task.json]: 1.0.4 is the latest "LTS" version of SDK

Comment (sachinma) [inline Tasks/DotNetCoreInstaller/task.json]: makes sense

Comment (sachinma) [inline Tasks/DotNetCoreInstaller/task.json]: Port it to master as well

**Pull request merged**